vendor_blueprints/refresh_datasource.py

```diff
@@ -0,0 +1,87 @@
+import tableauserverclient as TSC
```

Comment:
You should rename the directory `vendor_blueprints` (from the repo template) to the vendor's actual name, so `tableau_blueprints`